### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by cmdrkotori** [mpc-qt/mpc-qt#166] [General]
I presently plan to bundle a release at the end of June (xx.06) and end of December (xx.12) every year.

**Comment by tbertels** [comment]
Changelog draft for v24.12

- #138 and #139
- #140
- #144
- #151
- #152
- #153
- Increase the clickable area of the timeline (be267c4)
- #165
- #167
- #171
- #172
- #169
- #173
- #175 and #188
- Fix crash caused by native file dialog of Qt6 on macOS Ventura (and maybe Sonoma too) (7a3fd1a)
- #182
- #184
- #185 and #192
- #189 and #191
- #190
- #196
- #197
- #198
- #203
- #204
- #207
- #209
- #210
- Use a lock file to ensure single instance mode works correctly (1f88fb3, 7d5f30e, ea15e06)
- Display the title in the information bar nand fix chapter display (d2c16b1, 29a32fe)
- Add OSD font and font size settings (54c933e, 6470dc3)
- #225
- Only remember history for videos by default (e53bea7, 711eeba)
- #232
- #237

**Comment by tbertels** [comment]
While the 24.12 release was in early December, we could move the releases to early January and early July, to avoid having a xx.12 release that feels stale after less than a month.
They could be made ready by the end of the previous month but released a few days later.

**Reply by cmdrkotori** [reply]
I see no reason why not.

**Comment by tbertels** [comment]
I've created the 26.01 release branch.
Note to self:
* Notify translators on Weblate ("MPC-QT xx.xx will be released in early [Month], you have until then to update translations.  Thanks!")
* Modify .github/actions/data/VERSION
* Backport translations
* Add [version]-master tag to a commit unique to master.
* Update the AppStream metainfo file before tagging the release.
* Update direct links on https://mpc-qt.github.io/downloads
